src/client/containers/DeleteBoardModal/DeleteBoardModal.container.js

```diff
@@ -5,17 +5,25 @@ import CloseButton from '../../components/CloseButton/CloseButton.component';
 import PageTitle from '../../components/PageTitle/PageTitle.component';
 import GenericButton from '../../components/GenericButton/GenericButton.component';
 
-const DeleteBoardModal = ({ boardInfo, setModalVisibility }) => {
-  const userId = 2;
-  const boardId = 1;
+const DeleteBoardModal = ({
+  boardInfo,
+  setModalVisibility,
+  userId,
+  deleteErrorHandler,
+}) => {
+  const boardId = boardInfo.id;
+
   const deleteBoard = async () => {
     const API_URL = `/api/users/${userId}/boards/${boardId}`;
     try {
-      await fetch(API_URL, { method: 'DELETE' });
+      const response = await fetch(API_URL, { method: 'DELETE' });
       // eslint-disable-next-line no-alert
+      if (!response.ok) {
+        throw new Error(response.status);
+      }
       alert('Data deleted');
     } catch (error) {
-      throw new Error(error);
+      deleteErrorHandler(error);
```

Comment:
I don't see why you need to pass down a deleteHandler here.
You need to throw throw an ApiError in the catch function if its a general error that is not be handled in this component